frontend/src/components/ModalLink.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import Button from './Button'
+import styled from 'styled-components'
+import { Link, useHistory } from 'react-router-dom'
+
+const Directinglink = styled.a`
+  font-family: Lato;
+  font-size: 14px;
+  line-height: 16.8px;
+  text-decoration-line: underline;
+  color: #4d6daa;
+  text-align: center;
+  padding-bottom: 10px;
+  position: relative;
+  top: -15px;
+`
+
+type Props = {
+  text?: string
+  onClick: React.MouseEventHandler<HTMLElement>
+}
+
+function ModalLink(prop: Props) {
```

Comment:
this could be done in button; as a tertiary button - ie if state = 'tertiary', this is the button that renders

Reply:
Resolved in commit f24c30e8b814ee8bc691b9803db23a6fc53234a6.